features/fixtures/mazerunner/cxx-scenarios-bugsnag/src/main/cpp/cxx-scenarios-bugsnag.cpp

```diff
@@ -379,3 +379,17 @@ Java_com_bugsnag_android_mazerunner_scenarios_UnhandledExceptionWithUsageScenari
 }
 
 }
```

Comment:
The new function needs to appear before this semi-colon.

Reply:
It doesn't actually make a difference since the function is extern "C" already, but I've moved it for consitency.